src/Languages/Twig/Patterns/TwigRegexPattern.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Highlight\Languages\Twig\Patterns;
+
+use Tempest\Highlight\IsPattern;
+use Tempest\Highlight\Pattern;
+use Tempest\Highlight\Tokens\TokenType;
+
+final class TwigRegexPattern implements Pattern
```

Comment:
Couldn't we call this something like `TwigTokenPattern` or something alike?

I'd prefer it to be a bit more specific than "regex". But it's only a minor detail.